README.md

```diff
@@ -1,44 +1,225 @@
-# Open Source Project Template
+# requests_signature_dart
 
-This repository contains a template to seed a repository for an Open Source
-project.
+Signs and validates HTTP requests through Dio interceptors.
```

Comment:
You need to add more explanation of the recipe and mention the server-side implementation by referring to the .NET repository and explaining that the dart implementation is only relevant for the client-side part of the recipe.